web/src/components/common/SideBar.jsx

```diff
@@ -1,61 +1,46 @@
 import React, { Component } from 'react';
-import UploadByLink from '../views/CreateProjectView';
-import UploadByPattern from '../views/CreateProjectByPattern';;
-import ProjectContainer from '../views/ProjectContainer';
+import { Link } from 'react-router-dom';
+import logo from '../../style/img/white_logo_title.svg';
 
-class SearchBar extends Component {
+export default class SideBar extends Component {
   constructor(props) {
     super(props);
 
     this.state = {
       menu: {
         discoverProjects: 'Discover Projects',
         createProject: 'Create New Project',
+        bookmarks: 'Bookmarks',
+        profile: 'Profile',
         adminArea: 'Admin Area'
       }
     };
   }
 
-  render() {
+  render({ location }) {
```

Comment:
You're passing a string but it's expecting an object here. So no curly brackets.